api.js

```diff
@@ -0,0 +1,106 @@
+// use of API level -350
+
+const header = document.getElementById("header");
+const searchBox = document.querySelector("#search-shows");
+const paragraphforSearch = document.querySelector("#paragraphforSearch");
+const content = document.querySelector(".content");
+const selectShow = document.querySelector("#select-show");
+const selectEpisode = document.querySelector("#select-episode");
+let allEpisodes;
+const allShows = getAllShows();
+// console.log(title)
+
+loadShowOnPage(allShows);
+// popping up the episode in the window while it loads
+function loadShowOnPage(ShowList) {
+  header.innerHTML = "";
+  ShowList.sort((a, b) => a.name.localeCompare(b.name));
+  loadShow(ShowList);
+}
+
+// creating function for shows
+function loadShow(shows) {
+  for (let show of shows) {
+    const title = document.createElement("h2");
+    const image = document.createElement("img");
+    const paragraph = document.createElement("p");
+    const titleDiv = document.createElement("div");
+    const newbody = document.createElement("div");
+    //  adding episod name
+    title.innerText = `${show.name}`;
+    // title.appendChild(span)
+    titleDiv.appendChild(title);
+    // titleDiv.appendChild(span);
+    newbody.appendChild(titleDiv);
+    header.appendChild(newbody);
+    // episod image
+    console.log(show.image);
+    image.src =
+      show.image == null
+        ? "https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/summer-flowers-star-flower-1648071187.jpg"
+        : show.image.original;
+
+   newbody.appendChild(image);
+    header.appendChild(newbody);
+    image.classList.add("img");
+
+    // paragraph
+    paragraph.innerHTML = show.summary;
+    newbody.appendChild(paragraph);
+    header.appendChild(newbody);
+    newbody.classList.add("content");
+    titleDiv.classList.add("title-class");

```

Comment:
you could even extract this out into its own function (maybe called something like` createShowCard(show)`) - this means it could be reusable throughout other bits of your code, for example when selecting an episode from the episode dropdown you could use this function passing in the selected episode